docs/Network.md

```diff
@@ -50,6 +77,22 @@ fetch('https://mywebsite.com/endpoint.php')
   });
 ```
 
+2.  Called within an asynchronous function using ES7 `async`/`await` syntax:
```

Comment:
Cool, just wasn't sure whether ES7 was enabled in the Babel config by default.

Just found it :)
https://github.com/facebook/react-native/blob/master/packager/react-packager/.babelrc#L18